docs/pages/installation.rst

```diff
@@ -40,18 +40,3 @@ and make sure to include at least its core dependencies, which are:
     cp.read('../setup.cfg')
     print('.. code-block::\n')
     print(textwrap.indent(cp['options']['install_requires'].strip(), ' '))
-
-Legacy considerations
----------------------
-
-A legacy library titled ``uavcan`` (note the lack of the ``py`` prefix) is also available from PyPI,
-which implements an early experimental version of the protocol known as UAVCAN v0
-that is no longer recommended for new designs.
-It should not be confused with this library which implements the long-term stable version
-of the protocol known as UAVCAN v1.
-
-Having both ``pyuavcan`` and the old ``uavcan`` libraries installed in the same environment is not recommended,
-because PyUAVCAN generates Python packages from DSDL namespaces,
-and since the standard DSDL types are stored in the root namespace named ``uavcan``,
-it would conflict with the old library.
-Hence, when installing PyUAVCAN, make sure you don't have the legacy library around: ``pip uninstall -y uavcan``.
```

Comment:
It is possible that someone still has the `uavcan` package from v0 installed. I suggest that you replace this with a short note saying that that package will indeed conflict, but users can install `pyuavcan_v0` to mitigate this.

Reply:
I pushed a new (and last, presumably) update to the legacy `uavcan` package that removes the actual module from the system when updated, so it doesn't conflict anymore. In the worst case, we will provide guidance on the forum if there are still users affected by this.

Reply:
Alright.